emeis/conftest.py

```diff
@@ -62,9 +60,3 @@ def client(db):
 @pytest.fixture(scope="function", autouse=True)
 def _autoclear_cache():
     cache.clear()
-
-
-@pytest.fixture(autouse=True)
-def _remove_bootstrap_data(db):
-    for model in [User, Scope, Role, ACL]:
-        getattr(model, "objects").all().delete()
```

Comment:
This fixture has no usage and failed the linter because of proterty access by getattr.

Reply:
Careful, this is used implicitly! see `@pytest.fixture(autouse=True)`.

It's required because some old DB migration actually created some data.

The `getattr()` call could easily be replaced by `model.objects.all().delete()` to avoid triggering the linter

Reply:
Aaah no, I just remembered that I've moved the bootstrap data creation into a management command. In this case, you're absolutely right, this fixture is not required anymore and should have been removed back then

See https://github.com/projectcaluma/emeis/pull/266